cmd/containerboot/main.go

```diff
@@ -132,6 +132,16 @@ import (
 	"tailscale.com/util/linuxfw"
 )
 
+// exitCodeError is an error that includes the exit code from tailscaled.
+type exitCodeError struct {
```

Comment:
I think it's overcomplicating the requirements to pass tailscaled's exit code all the way back to containerboot's main. Let's just log tailscaled's exit code and pass a vanilla descriptive error back via the reaper channel instead.